client/utils/assignments.js

```diff
@@ -7,8 +7,10 @@ const isNotLockRestricted = (assignment, session) => (
         lockUtils.isItemLockedInThisSession(assignment, session)
 );
 
-const isTextAssignment = (assignment) => (
-    get(assignment, 'planning.g2_content_type') === PLANNING.G2_CONTENT_TYPE.TEXT
+const isTextAssignment = (assignment, contentTypes = []) => (
+    get(assignment, 'planning.g2_content_type') === PLANNING.G2_CONTENT_TYPE.TEXT ||
```

Comment:
`PLANNING.G2_CONTENT_TYPE.TEXT` points to a translation for text, so this might not work as expected in other languages.

Also I believe checking `contentTypes` only would be a better way to go, as the CV entry for text could be changed by an administrator.